README.md

```diff
@@ -119,6 +120,74 @@ const obj = {
 console.log(stringify(obj)) // '{"mail":"mail@example.com"}'
 ```
 
+#### Pattern properties
+`fast-json-stringify` supports pattern properties as defined inside JSON schema.  
+*patternProperties* must be an object, where the key is a valid regex and the value is an object, declared in this way: `{ type: 'type' }`.  
+*patternProperties* will work only for the properties that are not explicitly listed in the properties object.  
+Example:
+```javascript
+const stringify = fastJson({
+  title: 'Example Schema',
+  type: 'object',
+  properties: {
+    nickname: {
+      type: 'string'
+    }
+  },
+  patternProperties: {
+    'num': {
+      type: 'number'
+    },
+    '.*foo$': {
+      type: 'string'
+    }
+  }
+})
+
+const obj = {
+  nickname: 'nick',
+  matchfoo: 42,
+  otherfoo: 'str'
+  matchnum: 3
+}
+
+console.log(stringify(obj)) // '{"nickname":"nick","matchfoo":"42","otherfoo":"str","matchnum":3}'
+```
+`Object` and `Array` will be stringified as `{}` and `[]`.  
+Since `fast-json-stringify` is faster than `JSON.stringify` because works on a **fixed schema**, we strongly advice you to not pass complex objects or array as object values, but decompose them in their individual properties and stringify them.  
```

Comment:
Also, this sentence is probably not ok. It's perfectly fine to pass complex objects, as long as the schema stays fixed.